jf_agent/main.py

```diff
@@ -392,23 +394,48 @@ def generate_manifests(config, creds):
     # Create and add Jira Manifest
     if config.jira_url:
         agent_logging.log_and_print(logger, logging.INFO, 'Attempting to generate Jira Manifest...')
-        jira_manifest = create_jira_manifest(company_slug=company_slug, config=config, creds=creds)
-        if jira_manifest:
-            agent_logging.log_and_print(logger, logging.INFO, 'Successfully created Jira Manifest')
-            manifests.append(jira_manifest)
-        else:
+        try:
+            jira_manifest = create_jira_manifest(
+                company_slug=company_slug, config=config, creds=creds
+            )
+            if jira_manifest:
+                agent_logging.log_and_print(
+                    logger, logging.INFO, 'Successfully created Jira Manifest'
+                )
+                manifests.append(jira_manifest)
+            else:
+                agent_logging.log_and_print(
+                    logger, logging.ERROR, 'create_jira_manifest returned a None Type.'
+                )
+        except Exception as e:
             agent_logging.log_and_print(
-                logger, logging.ERROR, 'create_jira_manifest returned a None Type.'
+                logger,
+                logging.ERROR,
+                f'Error encountered when generating jira manifest. Error: {e}',
             )
     else:
         agent_logging.log_and_print(
             logger, logging.INFO, 'No Jira config detected, skipping Jira manifest generation'
         )
 
     if config.git_configs:
-        agent_logging.log_and_print(logger, logging.INFO, 'Attempting to generate Git Manifests...')
-        # Create and add Git Manifests
-        manifests += create_git_manifests(company_slug=company_slug, creds=creds, config=config)
+        try:
+            agent_logging.log_and_print(
+                logger, logging.INFO, 'Attempting to generate Git Manifests...'
+            )
+            # Create and add Git Manifests
+            manifests += create_git_manifests(
+                company_slug=company_slug,
+                creds=creds,
+                config=config,
+                endpoint_git_instances_info=jellyfish_endpoint_info.git_instance_info,
+            )
+        except Exception as e:
+            agent_logging.log_and_print(
```

Comment:
NB: Consider adding `traceback.format_exc()` here, so that we have exact details on what's erroring